### ISSUE TITLE
Myfinder hard drive monitoring details not working

### ISSUE THREAD

**Issue by cobbettjp** [mydockfinder/mydockfinder#407]
The hard drive monitoring details menu in Myfinder does not work since it was introduced in version 1.8.8.

When clicking the icon in my finder no menu is shown. I have tried updating to 1.8.8.5 and it is still not working. 

I have seen another user in MyDockFinder Discord with the same problem so it is also not working for others.

Memory, CPU and Temperature monitoring details menus are working fine.

**Comment by mydockfinder** [comment]
硬盘有没有什么特殊的接口或者是其他特别的设置




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月18日(星期一) 上午10:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)






My finder 中的 HDD / SSD 监视详细信息菜单无法显示，因为它是在 1 . 8 . 8 版本中引入的。

当点击图标在我的 finder 没有菜单显示。我已经试过更新到 1 . 8 . 8 . 5它仍然没有工作。

我看到另一个用户在 Discord 有同样的问题，所以它也不适用于其他人。

内存， CPU 和温度的细节菜单，但是工作正常。

—
直接回复此邮件，在 GitHub 上查看或取消订阅.
您收到这封邮件是因为您订阅了这个帖子。留言 ID ***@***.***与&gt;

**Comment by cobbettjp** [comment]
There is no special interface or settings that I am aware of.

My main drive is Samsung 970 Evo Plus NVMe for Windows.
I am also using a Samsung 860 Evo, Samsung 960 Evo and Samsung 970 Pro 512 NVMe.

My Windows version is Windows 11 Pro 21H2 Build 22000.795

Please let me know if you need any further information.

**Comment by mydockfinder** [comment]
硬盘有没有组raid




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月18日(星期一) 中午11:18
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)






没有特殊的界面或设置，我知道。

我的主驱动器是 Samsung 970 Evo Plus NV Me for Windows 。
我也在使用三星 860 Evo 、三星 960 Evo 和三星 970 Pro 512 NV Me 。

我的 Windows 版本是 Windows 11 Pro 21H2 Build 22000.795

请让我知道如果你需要任何进一步的信息。

—
直接回复此邮件，在 GitHub 上查看或取消订阅.
你收到这个是因为你评论了。留言 ID ***@***.***与&gt;

**Comment by cobbettjp** [comment]
No none of the hard drives are using RAID.

**Comment by mydockfinder** [comment]
If there is time, I can debug it remotely to show you the problem. I'm not controlling your computer remotely, but using your computer as my debugging environment, I can't see the contents of your computer, but I can find the problem from the code. But you need port mapping software, I don't know what software you have there that you can use



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月18日(星期一) 中午11:26
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
No none of the hard drives are using RAID.
 
&mdash;

**Comment by cobbettjp** [comment]
I dont have any remote debugging software or much time to set it up this evening unfortunately.

Let me know if I can provide any more information about my system that might help though.

**Comment by mydockfinder** [comment]
我可能需要一些硬盘的信息，系统自带的磁盘管理就可以了不过想要彻底修复我还是需要调试才能解决的，因为我这里磁盘是没问题的，所以我不能找到问题的根源




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月18日(星期一) 中午11:57
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
I dont have any remote debugging software or much time to set it up this evening unfortunately.
 
Let me know if I can provide any more information about my system that might help though.
 
&mdash;

**Comment by cobbettjp** [comment]
Yes the hard drive monitoring details are still not working for me on MyDockFinder 1.9 and 22H2.

What information do you need about the drives from Disk Management?

**Comment by mydockfinder** [comment]
是所有磁盘都没有用吗，还是某一个磁盘
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月12日(星期三) 上午7:10
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)






是的，硬盘监控细节在MyDockFinder 1.9和22H2上仍然不起作用。

您需要有关“磁盘管理”中驱动器的哪些信息？

—
直接回复此电子邮件，在 GitHub 上查看或取消订阅。
您收到此消息是因为您发表评论。Message ***@***.***与&gt;

**Comment by cobbettjp** [comment]
MyDockFinder is installed to my main drive which is a Samsung 970 Evo Plus NVMe. I also have the three other drives I mentioned previously. 

I could try moving the MyDockFinder installation to my secondary Steam Library on a different hard drive. This might rule out whether the problem is specific to the drive or general to the system.

**Comment by mydockfinder** [comment]
能不能拍一个视频，我想看一下详细情况



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月12日(星期三) 上午8:49
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
MyDockFinder is installed to my main drive which is a Samsung 970 Evo Plus NVMe. I also have the three other drives I mentioned previously.
 
I could try moving the MyDockFinder installation to my secondary Steam Library on a different hard drive. This might rule out whether the problem is specific to the drive or general to the system.
 
&mdash;

**Comment by cobbettjp** [comment]
Yes that fine, what do you want the video to show?

**Comment by mydockfinder** [comment]
就是正常运行的时候，点击myfinder磁盘，显示详细信息的操作主要是展示出现的问题




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月12日(星期三) 上午9:04
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
Yes that fine, what do you want the video to show?
 
&mdash;

**Comment by cobbettjp** [comment]
https://www.dropbox.com/s/fb4assoltkfr6cd/2022_10_12_021454.mp4?dl=0

**Comment by mydockfinder** [comment]
你这个问题不是很好办，应该是某个磁盘出现了问题，抛出了异常所以就不会显示菜单了我会在下个版本修改一下，如果是某个磁盘异常就会跳过这个磁盘不显示他




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月12日(星期三) 上午9:26
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
https://www.dropbox.com/s/fb4assoltkfr6cd/2022_10_12_021454.mp4?dl=0
 
&mdash;

**Comment by cobbettjp** [comment]
Okay thanks for your help. I will check to see if it's working in the next version.

**Comment by mydockfinder** [comment]
不客气




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月12日(星期三) 上午9:40
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Myfinder hard drive monitoring details not working (Issue #407)





 
Okay thanks for your help. I will check to see if it's working in the next version.
 
&mdash;
